src/main/java/reposense/model/FileTypeManager.java

```diff
@@ -48,7 +53,14 @@ private FileType getFileFormat(String fileName) {
                     "This exception should not happen as we have performed the whitelisted formats check.");
         } else {
             String[] tok = fileName.split("[./\\\\]+");
-            return FileType.convertStringFormatToFileType(tok[tok.length - 1]);
+            String deducedFormat = tok[tok.length - 1];
```

Comment:
`deducedFormat` could be better described as `extension` or `label` or `format`.